src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -811,10 +811,11 @@ public bool IsFrozen
             }
             set
             {
+                bool oldValue = isFrozenExplicitly;
```

Comment:
why this is not an issue in 1.3?

Reply:
@ramramps Because we changed how `Freeze state` got deserialized, the property is now set from extra blob.